sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -657,16 +657,17 @@ def _build_item(self, item):
         return item
 
 
-class FilteredBlob(FilterBlobItem):
+class FilteredBlob(DictMixin):
     """Blob info from a Filter Blobs API call.
 
     :ivar name: Blob name
     :type name: str
     :ivar container_name: Container name.
     :type container_name: str
-    :ivar tag_value: tag value filtered by the expression.
-    :type tag_value: str
```

Comment:
this is to address Kamil's comment. There's a bug on service side so that only one tag value is returned, we don't want to expose tag_value currently